doc/releases/changelog-dev.md

```diff
@@ -172,6 +172,13 @@
 * The module `pennylane/math/quantum.py` has now support for the min-entropy.
   [(#3959)](https://github.com/PennyLaneAI/pennylane/pull/3959/)
 
+* A function called `apply_operation` has been added to the new `qutrit_mixed` module found in `qml.devices` that applies operations to device-compatible states.
```

Comment:
Sorry if I missed this in a previous discussion thread, but why is this showing up as a new change copied down here? It looks like it appears in two places in the file now, but the update for `measure` is only in this section.

Reply:
I think there was an error in merging I tried to fix it by deleting some copying I will revert this file to the error and fix it.